Applications/ApplicationsLib/LinearSolverLibrarySetup.h

```diff
@@ -19,7 +19,7 @@
 /// The default implementation is empty providing polymorphic behaviour when
 /// using this class.
 
-#include "MathLib/LinAlg/GlobalMatrixProviders.h"
+#include "AssemblerLib/GlobalMatrixProviders.h"
```

Comment:
This issue can be fixed IMHO in a better way by consistently naming the member functions of the DOF table. Then the DOF related methods of the matrix and vector providers would only get integer arguments and not the whole DOF table.